src/VideoRemise/VideoRemise/HelpPage.xaml

```diff
@@ -0,0 +1,144 @@
+                </Paragraph>
+                <Paragraph Margin="20,0,0,0">
+                    <InlineUIContainer>
+                        <SymbolIcon Symbol="Stop"/>
+                    </InlineUIContainer> stops recording the current match and saves the video recordings if a match
+                had been set up.
+                </Paragraph>
+                <Paragraph Margin="20,0,0,0">
+                    <InlineUIContainer>
+                        <SymbolIcon Symbol="Setting"/>
+                    </InlineUIContainer> brings up the Settings page, which allows you to select camera devices and
+                configure your scoring machine.
+                </Paragraph>
+                <Paragraph Margin="20,0,0,0">
+                    <InlineUIContainer>
+                        <SymbolIcon Symbol="Help"/>
+                    </InlineUIContainer> displays this help screen.
+                </Paragraph>
+                <Paragraph FontSize="18" FontWeight="Bold" Margin="0,5,0,0">Basic Flow</Paragraph>
+                <Paragraph>
+                    VideoRemise starts in Idle state. In this mode, it displays the live feed from
+                each camera on the main video display. No video is recorded.
+                </Paragraph>
+                <Paragraph Margin="0,5,0,0">
+                    When you start a match using the Start command bar button,
+                    <InlineUIContainer>
+                        <SymbolIcon Symbol="Camera"/>
+                    </InlineUIContainer> ,
+                VideoRemise transitions to the Recording state. In this mode it continues to 
+                display the live feed, but it is now recording the feed as well.
+                </Paragraph>
+                <Paragraph Margin="0,5,0,0">
+                    When a trigger occurs, either a hit signalled by the scoring machine or a
+                hit or halt signalled manually (see below), VideoRemise enters the Replaying
+                state. In this mode, it will replay the last action (a few seconds before and
+                after the trigger) in a continuous loop.
+                </Paragraph>
+                <Paragraph Margin="0,5,0,0">
+                    While in the Replaying mode, VideoRemise continues to record the feed and look
+                for triggers. If a trigger occurs while replaying, VideoRemise will stop
+                replaying the previous action and start replaying the new action.
+                </Paragraph>
+                <Paragraph FontSize="18" FontWeight="Bold" Margin="0,5,0,0">Manual Triggers</Paragraph>
+                <Paragraph>
+                    In Recording or Replaying mode, the following keys may be used to trigger the system:
+                </Paragraph>
+                <Paragraph Margin="20,0,0,0">
+                    <Bold>Return</Bold> signals a Halt with no lights; for instance, the referee has called Halt
```

Comment:
May want to use enter/return or something like that, not sure how many people will know return=enter.

Reply:
Done.